mautic/mautic
# Removing ExceptionHelper as it is not used anywhere

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [Y]
| New feature/enhancement? (use the a.x branch)      | [N]
| Deprecations?                          | [N]
| BC breaks? (use the c.x branch)        | [Y]
| Automated tests included?              | [N (removing a file)] 
| Related user documentation PR URL      | /
| Related developer documentation PR URL | Added to `UPGRADE-5.0.md`
| Issue(s) addressed                     | Fixes https://github.com/mautic/mautic/pull/11145



#### Description:

While working on https://github.com/mautic/mautic/pull/11145 I found that ExceptionHelper is using deprecated `%kernel.root_dir%` param as a dependency but this service is not used anywhere, so it's safe to remove it for M5 as it cannot be tested anyway.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. You can hack a file that will create an error. For example add a typo to the app/AppKernel.php file and check that the classic Mautic error page with Mautibot appears.
